### ISSUE TITLE
[Bug]: Docker image should have `micromamba`, but not `mamba`

### ISSUE THREAD

**Issue by mfisher87** [nsidc/qgreenland#564]
### Issue description

I'm not sure what is causing the issue that was resolved by #547 . We use `micromamba` to install environments, so we shouldn't ever need `mamba` in the Docker image, especially at runtime. I'm so confused about that... I think we need to look closer at whatever is causing that error!

TODO: Reproduction instructions for error in description of #547

### System information

N/A

**Comment by mfisher87** [comment]
@trey-stafford I spent a minute trying to replicate this. I:

* Removed the line from `Dockerfile` which installs `mamba`
* Built the image
* `docker-compose up`.

The Luigi scheduler successfully started with no error. Is there something else I should be trying to replicate the error?

**Comment by trey-stafford** [comment]
Perhaps we can just remove the line that installs mamba. To replicate, the step that installs `mamba` has to come _before_ the step that installs dependencies from conda lock. Revert the changes in #547 to replicate:

```
$ docker compose up
[+] Building 0.0s (0/0)
[+] Running 3/3
 ✔ Network qgreenland_default  Created                                                                                                                                                                                                                                                                                                            0.1s
 ✔ Container luigi             Created                                                                                                                                                                                                                                                                                                            0.0s
 ✔ Container webserver         Created                                                                                                                                                                                                                                                                                                            0.0s
Attaching to luigi, webserver
webserver  | /docker-entrypoint.sh: /docker-entrypoint.d/ is not empty, will attempt to perform configuration
webserver  | /docker-entrypoint.sh: Looking for shell scripts in /docker-entrypoint.d/
webserver  | /docker-entrypoint.sh: Launching /docker-entrypoint.d/10-listen-on-ipv6-by-default.sh
webserver  | 10-listen-on-ipv6-by-default.sh: info: can not modify /etc/nginx/conf.d/default.conf (read-only file system?)
webserver  | /docker-entrypoint.sh: Launching /docker-entrypoint.d/20-envsubst-on-templates.sh
webserver  | /docker-entrypoint.sh: Launching /docker-entrypoint.d/30-tune-worker-processes.sh
webserver  | /docker-entrypoint.sh: Configuration complete; ready for start up
luigi      | Traceback (most recent call last):
luigi      |   File "/opt/conda/bin/mamba", line 7, in <module>
luigi      |     from mamba.mamba import main
luigi      | ModuleNotFoundError: No module named 'mamba'
luigi exited with code 0
luigi      | Traceback (most recent call last):
luigi      |   File "/opt/conda/bin/mamba", line 7, in <module>
luigi      |     from mamba.mamba import main
luigi      | ModuleNotFoundError: No module named 'mamba'
luigi exited with code 0
luigi      | Traceback (most recent call last):
luigi      |   File "/opt/conda/bin/mamba", line 7, in <module>
luigi      |     from mamba.mamba import main
luigi      | ModuleNotFoundError: No module named 'mamba'
luigi exited with code 1
luigi      | Traceback (most recent call last):
luigi      |   File "/opt/conda/bin/mamba", line 7, in <module>
luigi      |     from mamba.mamba import main
luigi      | ModuleNotFoundError: No module named 'mamba'
luigi exited with code 1
luigi      | Traceback (most recent call last):
luigi      |   File "/opt/conda/bin/mamba", line 7, in <module>
luigi      |     from mamba.mamba import main
luigi      | ModuleNotFoundError: No module named 'mamba'
luigi exited with code 1

```

**Comment by mfisher87** [comment]
Got it. Yes, I think removing the line that installs `mamba` and `conda` is the solution also. I don't know why it was there in the first place :) Must be vestigial from using a much earlier version of micromamba, or perhaps before we started using micromamba?

I also don't know why the container would look for mamba at start-up time... maybe when we installed mamba some setup scripts or alterations to `.bashrc` were applied, and then when we bootstrapped with `conda-lock`, we blew away `mamba`? :shrug:
